resqpy/model.py

```diff
@@ -2912,13 +2912,10 @@ def iter_wellbore_interpretations(self):
       """
       import resqpy.organize  # Imported here for speed, module is not always needed
 
-      # TODO: instantiate using UUIDs
-      parts = self.parts_list_of_type('WellboreInterpretation')
-      for part in parts:
-         well_root = self.root_for_part(part)
-         well = resqpy.organize.WellboreInterpretation(self, root_node=well_root)
-         yield well
-
+      uuids = self.uuids(obj_type='WellboreInterpretation')
```

Comment:
I believe that Model.uuids(), parts(), roots() and titles() should now be guaranteed to return a list (possibly empty), so it should be safe to remove the `if uuids:` here.